### PR TITLE
Clpeterson/fix/insufficient funds gas

### PR DESCRIPTION
**Pull request by CLPeterson**

### CODE REVIEW

**Review by rcresnik — APPROVED** [review]

**Comment by kenanmamedoff** [comment]
@rcresnik is right @CLPeterson. we need these changes on the main branch

File: brd-ios/breadwallet/src/ViewControllers/RootModals/SendViewController.swift [hunk 1]
```diff
@@ -328,13 +328,27 @@ class SendViewController: UIViewController, Subscriber, ModalPresentable {
     }
     
     // TODO: WalletKit is not throwing the right error. Instead, we are always getting "WalletKit.Wallet.FeeEstimationError.ServiceError" error.
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/ViewControllers/RootModals/SendViewController.swift:330]: This comment can be removed

File: brd-ios/breadwallet/src/ViewControllers/RootModals/SendViewController.swift [hunk 2]
```diff
@@ -328,13 +328,27 @@ class SendViewController: UIViewController, Subscriber, ModalPresentable {
...
+                        _ = handleValidationResult(.insufficientFunds)
+                    } else {
+                        _ = handleValidationResult(.insufficientGas)
+                    }
+                } else {
+                    _ = handleValidationResult(.insufficientFunds)
+                }
+            } else {
+                _ = handleValidationResult(.insufficientFunds)
+            }
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/ViewControllers/RootModals/SendViewController.swift:351]: https://bayes.atlassian.net/browse/RW-516

I don't know if it already does but this piece of code should behave like it was described in this ticket.

File: brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift [hunk 3]
```diff
@@ -281,6 +281,8 @@ class ImportKeyViewController: UIViewController, Subscriber {
         switch error {
         case .InsufficientFunds:
             showErrorMessage(L10n.Send.insufficientFunds)
+        case .InsufficientGas:
+            showErrorMessage(L10n.Send.insufficientFunds)
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift:285]: should InsufficientFunds and InsufficientGas have the same messages? then we can put these two enum cases together.

**Pull request closed without merging**